snippets/fuzzySearch.md

```diff
@@ -0,0 +1,19 @@
+### fuzzySearch
```

Comment:
I believe we definitely need to change  the snippet name from `fuzzySearch` to something else like the following 

`lazySearch`
`lazyMatch`
`isSimilar`

To be honest I am currently indifferent as to the exact name & okay with whatever we agree on